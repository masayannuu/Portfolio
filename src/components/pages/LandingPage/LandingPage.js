import React from 'react'
import LandingPageTemplate from '../../templates/LandingPage/LandingPage'

const LandingPage = () => <LandingPageTemplate />

export default LandingPage
