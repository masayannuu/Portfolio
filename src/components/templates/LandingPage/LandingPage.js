import React from 'react'
import MediaQuery from 'react-responsive'
import { Image } from 'reactbulma'

import { Head, Profile, ThreePoints, Work, Sns, Footer } from '../../organisms/index'
import { Rain, Arrow, Bubble } from '../../atoms/Animation/index'
import Umbrella from '../../../../public/umbrella.png'

const LandingPage = () => (
  <div>
    <Bubble />
    <Head />
    <MediaQuery query="(min-width: 1701px)">
      <Rain targetY={900} isMobile={false} />
    </MediaQuery>
    <MediaQuery query="(min-width: 1282px) and (max-width: 1700px)">
      <Rain targetY={800} isMobile={false} />
    </MediaQuery>
    <MediaQuery query="(min-width: 769px) and (max-width: 1281px)">
      <Rain targetY={680} isMobile={false} />
    </MediaQuery>
    <MediaQuery query="(max-width: 768px)">
      <Rain targetY={200} isMobile />
    </MediaQuery>
    <Image src={Umbrella} is="128x128" />
    <Profile />
    <MediaQuery query="(min-width: 1701px)">
      <Arrow targetY={1700} />
    </MediaQuery>
    <MediaQuery query="(min-width: 1282px) and (max-width: 1700px)">
      <Arrow targetY={1600} />
    </MediaQuery>
    <MediaQuery query="(min-width: 769px) and (max-width: 1281px)">
      <Arrow targetY={1550} />
    </MediaQuery>
    <MediaQuery query="(max-width: 768px)">
      <Arrow targetY={1500} />
    </MediaQuery>
    <ThreePoints />
    <Work />
    <Sns />
    <Footer />
  </div>
)

export default LandingPage
