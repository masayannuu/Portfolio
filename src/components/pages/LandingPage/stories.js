import React from 'react'
import { storiesOf } from '@storybook/react'

import LandingPage from './LandingPage'

storiesOf('Pages.LandingPage', module).add('LandingPage', () => <LandingPage />)
