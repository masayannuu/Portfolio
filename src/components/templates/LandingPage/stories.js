import React from 'react'
import { storiesOf } from '@storybook/react'

import LandingPage from './LandingPage'

storiesOf('Templates.LandingPage', module).add('LandingPage', () => <LandingPage />)
