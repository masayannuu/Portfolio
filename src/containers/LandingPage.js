import React from 'react'
import LandingPage from '../components/pages/LandingPage/LandingPage'

export default () => <LandingPage />
