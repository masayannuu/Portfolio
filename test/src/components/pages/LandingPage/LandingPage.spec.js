import React from 'react'
import Enzyme from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'

import LandingPage from '../../../../../src/components/pages/LandingPage/LandingPage'

describe('LandingPage LandingPage Molecules', () => {
  it('should render LandingPage', () => {
    const wrapper = Enzyme.shallow(<LandingPage />)
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })
})
