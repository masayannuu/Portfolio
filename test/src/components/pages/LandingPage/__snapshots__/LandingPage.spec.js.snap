// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingPage LandingPage Molecules should render LandingPage 1`] = `<LandingPage />`;
